specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/billingSubscription.json

```diff
@@ -612,131 +681,545 @@
+        "productTypeId": {
+          "description": "Id of the product for which the subscription is purchased",
+          "type": "string"
         },
-        "invoiceSectionDisplayName": {
-          "description": "The name of the invoice section to which the subscription is billed.",
+        "purchaseDate": {
           "type": "string",
+          "format": "date-time",
+          "description": "Purchase date of the product in UTC time.",
           "readOnly": true
         },
+        "quantity": {
+          "description": "The number of licenses purchased for the subscription",
+          "type": "integer",
+          "format": "int64"
+        },
         "reseller": {
-          "description": "Reseller for this subscription.",
+          "description": "Reseller for this subscription. The fields is not available for Microsoft Partner Agreement billing accounts.",
           "readOnly": true,
           "$ref": "#/definitions/Reseller"
         },
+        "renewalTermDetails": {
+          "description": "Renewal term details of the product.",
+          "type": "object",
+          "$ref": "#/definitions/RenewalTermDetails"
+        },
+        "skuDescription": {
+          "description": "The SKU description of the product for which the subscription is purchased. This field is only available  for Microsoft Customer Agreement billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
         "skuId": {
-          "description": "The sku ID of the Azure plan for the subscription.",
+          "description": "The SKU ID of the product for which the subscription is purchased. This field is only available  for Microsoft Customer Agreement billing accounts.",
           "type": "string"
         },
-        "skuDescription": {
-          "description": "The sku description of the Azure plan for the subscription.",
+        "status": {
+          "description": "The status of the subscription. This field is not available for Enterprise Agreement billing accounts.",
+          "type": "string",
+          "enum": [
+            "Unknown",
+            "Active",
+            "Disabled",
+            "Deleted",
+            "Warned",
+            "Expiring",
+            "Expired",
+            "AutoRenew",
+            "Cancelled",
+            "Suspended"
+          ],
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "BillingSubscriptionStatus",
+            "modelAsString": true
+          }
+        },
+        "subscriptionId": {
+          "description": "The ID of the usage-based subscription. This field is only available for usage-based subscriptions of Microsoft Customer Agreement Type billing accounts.",
           "type": "string",
           "readOnly": true
         },
         "suspensionReasons": {
-          "description": "The suspension reason for a subscription. Applies only to subscriptions in Microsoft Online Services Program billing accounts.",
```

Comment:
"The suspension reason for the subscription". We are using "the" everywhere else.